packages/schemas/tables/usersroles.sql

```diff
@@ -0,0 +1,5 @@
+create table users_roles (
```

Comment:
same for the filename? `users_roles.sql`

Reply:
discussed offline